pywinauto/unittests/test_application.py

```diff
@@ -1169,6 +1169,19 @@ def test_depth(self):
             len(self.app['Font'].descendants(depth=1)),
             len(self.app['Font'].descendants(depth=2)))
 
+    if UIA_support:
+        def test_child_window_depth(self):
+            """Test that child_window() with depth works correctly"""
+            # TODO fix same elements at different tree levels on win32 backend and switch test back to win32
+            _app = Application(backend="uia").start("Notepad")
```

Comment:
The application wasn't killed after the test. Usually we start the app in `setUp` method of the TestCase class and kill it in `tearDown` method. It gives us a guarantee to clean everything after the test. Maybe connect to existing app that was started in `setUp` method?

Reply:
Fixed at 1ce09c82. Test moved to separate class `UIAWindowSpecificationTestCases`